course-v2/layouts/partials/resources_header.html

```diff
@@ -7,14 +7,15 @@
   <h2>Download</h2>
   <div class="hide-offline download-course-container background-color-light-grey p-3">
     <div class="row">
-      <div class="d-flex justify-content-center col-3 px-3">
+      <div class="d-flex justify-content-center col-12 col-md-3 px-3">
         <a class="download-course-button p-2" href="{{ $downloadCourseUrl }}">
           <span class="material-icons">file_download</span> Download course
         </a>
       </div>
-      <div class="col-9 pl-0 pr-3">
-        This package contains the same content as the online version of the course, 
-        <em>except for the audio/video materials</em>. These can be downloaded below. For help 
+      <div class="download-mobile-margin"></div>
```

Comment:
This class isn't necessary. On the div above, you can simply add `pb-2` to add .5rem of padding. In this case the bottom padding doesn't affect the desktop style at all, but if you wanted to make sure there was no bottom padding on desktop you would simply add `pb-md-0`. Either way, if you were going to add a style like this you should use the `@include media-breakpoint-down` syntax so that the breakpoint coincides with how the rest of the site works.

Reply:
Updated in c4ae011a8298e60275a2a4d4223ac8a902f02b27.